eclass/texlive-common.eclass

```diff
@@ -55,9 +53,13 @@ texlive-common_handle_config_files() {
 
 texlive-common_is_file_present_in_texmf() {
 	local mark="${T}/$1.found"
-	[ -d texmf ] && find texmf -name $1 -exec touch "${mark}" \;
-	[ -d texmf-dist ] && find texmf-dist -name $1 -exec touch "${mark}" \;
-	[ -f "${mark}" ]
+	if [[ -d texmf ]]; then
+		find texmf -name ${1} -exec touch ${mark} \; || die
```

Comment:
You are not passing the filename to `touch`, so I'm wondering why this doesn't throw usage error… or does it?

Reply:
filename is above.

Reply:
(yes, I fixed {})